corehq/apps/accounting/tests/test_tasks.py

```diff
@@ -51,3 +56,111 @@ def test_calculate_form_submitting_mobile_workers_in_all_domains(self):
         self.assertEqual(worker_history.domain, self.domain.name)
         self.assertEqual(worker_history.num_users, self.num_form_submitting_workers)
         self.assertEqual(worker_history.record_date, self.one_day_ago)
+
+
+class TestSubscriptionReminderEmails(BaseInvoiceTestCase):
+    def setUp(self):
+        super().setUp()
+        toggles.SHOW_AUTO_RENEWAL.set(self.domain.name, True, namespace=toggles.NAMESPACE_DOMAIN)
+
+    def test_sends_renewal_reminder_email(self):
+        self.subscription.date_end = datetime.date.today() + datetime.timedelta(days=60)
```

Comment:
I'd feel more comfortable if we used freezegun to specify a hard date. While unlikely, I could see these tests returning failures if run on day boundaries. The subscription could be saved with one date, while `send_renewal_reminder_emails` could be called with another. Using freezegun would eliminate that possibility

Reply:
If this is meant to basically be set in the background (i.e. the tests don't care about the specific date), it probably is cleaner to use the [class decorator](https://time-machine.readthedocs.io/en/latest/usage.html#class-decorator)

Reply:
I did this initially, but found when I tried to set `tick=False` it would cause the tests to time out and fail. Not happening any more though, must have been something wonky in my local environment. https://github.com/dimagi/commcare-hq/pull/37072/commits/8b8eafd5e4e81943f2cba96ed2b4c817310ac327